pkgs/development/python-modules/decorator/default.nix

```diff
@@ -12,9 +14,11 @@ buildPythonPackage rec {
     sha256 = "54c38050039232e1db4ad7375cfce6748d7b41c29e95a081c8a6d2c30364a2ce";
   };
 
-  postPatch = ''
-    substituteInPlace src/tests/test.py --replace "DocumentationTestCase" "NoDocumentation"
-  '';
+  patches = lib.optional isPy27
```

Comment:
As discussed with @disassembler in IRC, let's merge this, we shouldn't block this many package builds much longer.

But I'm pretty sure this should be removed again here, I don't think this failure has anything to do with python 2. Not applying it to python 3 might result in future random build failures again.